recommendations/io-readmore.md

```diff
@@ -13,29 +13,20 @@ The profiled process has been observed is unusually idle under load, typically t
 it's waiting for external I/O because there's nothing else to do until the I/O completes.
 
 To solve I/O issues we have to track down the asynchronous call(s) which are taking an
-abnormally long time to complete.
+abnormally long time to complete. I/O root cause analysis is mostly a reasoning exercise.
 
-I/O root cause analysis is mostly a reasoning exercises, or requires advanced knowledge and
-expertise with specialist Node.js logging flags and (very new) asynchronous tracking API's.
-
-At nearForm we care a lot about this problem and we are developing a new tool to make
-I/O debugging easier... stay tuned!
+It used to requires advanced knowledge and
+expertise with specialist Node.js logging flags and (very new) asynchronous tracking API's – however, we care a lot about this problem at nearForm, and have developed a new Clinic tool to make I/O debugging easier: [Clinic Bubbleprof](https://clinicjs.org/bubbleprof)
 
 ## Next Steps
-- Use (or create) an external architecture diagram (logical, hardware, whatever is available) to
-understand all I/O "touch points", that is all I/O to/from the Node.js process (e.g. databases, network requests, filesystem...)
-- Measure the response times of each of the I/O touch points
-  - One approach is to instrument the Node.js process with `console.time` before an asynchronous call
-    and `console.timeEnd` at the top of the callback (or promise `then` handler, or after an `await` or whatever the asynchronous abstraction)
-  - Another approach is write benchmarks specifically for the I/O touch points
-    - Be sure to duplicate the exact conditions as generated in the Node.js process
-- Run these measurements multiple times, look for unreasonably high response times
-- Once slow I/O touch points have been discovered a strategy for speeding up the I/O is required  
-
-**Advanced**: For more advanced, lower overhead, timing functionality, check out the experimental `perf_hooks` API  
-
-**Advanced:** An alternative to the approach outlined above is to make use of the experimental
-Node.js `async_hooks` API, in combination with a timer and stack trace generation.
+- Make sure you are aware of the application's I/O "touch points": that is, I/O to and from the Node.js process (e.g. databases, network requests, filesystem...)
```

Comment:
can we avoid changing pronoun to 2nd person, all of the recommendation are written in 1st person plural + 3rd person omniscient